dartagnan/src/main/java/com/dat3m/dartagnan/program/event/Tag.java

```diff
@@ -117,8 +114,8 @@ private C11() {
         public static final String ATOMIC               = "A";
         public static final String NONATOMIC            = "NA";
 
-        public static final String PTHREAD              = "PTHREAD";
-        public static final String LOCK                 = "LOCK";
+        public static final String PTHREAD              = "__PTHREAD";
+        public static final String LOCK                 = "LK";
```

Comment:
I am not too sure about this (i.e. should be `__LK`?). LKMM does not use this tag (it splits the lock into the read an write parts `LKR, LKW`). In SVCOMP we use the tag as an internal one. [C11 mentions it, but we have that part of the model commented out](https://github.com/hernanponcedeleon/Dat3M/blob/abc74af479b39b3005ed75dbe5bd2757dcf3e64b/cat/c11.cat#L46). Also, it is not clear to me where this comes from in the original model. There is this `include c11_los.cat`, but [that one refers to `LS`](https://github.com/herd/herdtools7/blob/master/herd/libdir/c11_los.cat)